libretranslate/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: "3"
+
+services:
+  app_proxy:
+    environment:
+      # <app-id>_<web-container-name>_1
+      # e.g. 'btc-rpc-explorer_web_1'
+      # Note that the '_1' at the end is needed
+      APP_HOST: libretranslate_main_1
+      APP_PORT: 5000
+      PROXY_AUTH_ADD: "false"
+  main:
+    image: libretranslate/libretranslate:v1.5.6
```

Comment:
We need to add the multi-arch digest here to lock this image in to the exact tested image. Without the digest, if for some reason a new image is pushed with the same tag (e.g., `v1.5.6`) that accidentally breaks something then users will get this broken image on app restart or fresh install.

Reply:
Totally forgot about that here. All the other submissions (i think) have digests.